include/enums.h

```diff
@@ -173,7 +173,7 @@ namespace buildOrder {
 		REMOVEME
 	};
 
-    static const char* eBuildOrderStateString(const eBuildOrderState &state) {
+    inline const char* eBuildOrderStateString(const eBuildOrderState &state) {
```

Comment:
what is the difference between `static` and `inline` here?

Reply:
For free functions (so not member functions), static means that it has "internal linkage", which means it is only accessible from the current compilation unit. So it only makes sense for definitions in the .cpp file. As a side-note: putting symbols in an anonymous namespace has the same effect and is recommended above the static keyword.

The inline keyword means that multiple implementations of a function may exist and that the linker can just pick one. This is for functions with the implementation in the header file.